openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -120,8 +120,15 @@ def _update_tool_with_data(self, tool, data):
             return
 
         original_subset = tool.GetData("openpype.subset")
+        original_format = tool.GetData(
+            "openpype.creator_attributes.image_format"
+        )
+
         subset = data["subset"]
-        if original_subset != subset:
+        if (
+            original_subset != subset
+            or original_format != data["creator_attributes"]["image_format"]
```

Comment:
Do note that currently there's nothing actually updating `creator_attributes.image_format`. So it will always match the stored data because nothing exposes that attribute to the user, right? or does it?

Reply:
In the Publish menu currently you can see/edit.
![image](https://github.com/ynput/OpenPype/assets/20257044/7a462790-9c31-4580-9b5b-4981b43c1347)

The "Output Image Format" = creator_attributes.image_format. So if the user changes it in this menu it'll update not just the creator_attributes on the saver but also the path using `_configure_saver_tool`

Reply:
Actually, that is perfect then - yes! ❤️ Thanks for the screenshot.